### PR TITLE
fix woq example and update document for v1.19.0

### PR DESCRIPTION
**Pull request by xin3he**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not

## Description

detail description

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by thuang6 — APPROVED** [review]
LGTM

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py [hunk 1]
```diff
@@ -270,8 +270,9 @@ def get_user_model():
         torchscript = True
     if args.woq_algo == "AutoRound" and is_habana_framework_installed():
         print("Quantizing model with AutoRound on HPU")
-        check_torch_compile_with_hpu_backend()
-        set_envs_for_torch_compile_with_hpu_backend()
+        if args.quantize:
```
Comment (yiliu30) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:273]: Please attach the example test report for reference.

Comment (xin3he) [reply examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:273]: opt_125m_woq_autoround_int4: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/151/
opt_125m_woq_autoround_int4_hpu: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/152/

**Comment by xin3he** [comment]
image
it's a fix for accuracy, bf16 cannot get correct accuracy as below, no idea but bf16 is not necessary so remove it.
[inc_gaudi_test #150 [Jenkins]](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/150/)

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py (hunk 1 above)
Comment (yiliu30) [reply examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:273]: Both extension tests are running on the HPU. Is that expected?

**Comment by xin3he** [comment]
latest: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/153/

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py (hunk 1 above)
Comment (xin3he) [reply examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py:273]: Yes, here I intend to compare the impact of special env variables for autoround.

**Comment by xin3he** [comment]
@XuehaoSun  please help trigger the test of `opt_125m_woq_autoround_int4_hpu`, not `opt_125m_woq_autoround_int4` after removing environment variables and using `requirement-autoround-hpu.txt` in validation code.

**Comment by yiliu30** [comment]
@xin3he @XuehaoSun suggest testing the auto-round on the CPU as well.

**Comment by xin3he** [comment]
cpu: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/5595/

**Review by yiliu30 — APPROVED** [review]

**Comment by xin3he** [comment]
Verify complete GPTQ test based on https://github.com/intel/neural-compressor/pull/2100 
https://inteltf-jenk.sh.intel.com/job/inc_gaudi_test/156/

**Review by chensuyue — APPROVED** [review]

**Comment by xin3he** [comment]
The CPU test failed @XuehaoSun, please help verify the validation machine environment issue.

This issue is not related to this PR and this PR didn't change CPU related logic. so we merge it first for release test.  @yiliu30

**Comment by yiliu30** [comment]
> The CPU test failed @XuehaoSun, please help verify the validation machine environment issue.
> 
> This issue is not related to this PR and this PR didn't change CPU related logic. so we merge it first for release test. @yiliu30

@xin3he  It's fine with me. Please create a Jira to track it in case we miss it in the future.

**Pull request merged**